controllers/provider-aws/cmd/gardener-extension-provider-aws/app/app.go

```diff
@@ -66,6 +83,11 @@ func NewControllerManagerCommand(ctx context.Context) *cobra.Command {
 			MaxConcurrentReconciles: 5,
 		}
 
+		// // options for the controlplane controller
+		// backupBucketCtrlOpts = &controllercmd.ControllerOptions{
```

Comment:
Why is this here?

Reply:
I guess it came from old commit while rebase. I'll clear that. Strange it didn't give any error in `make verify`